locales/en/plugin__console-plugin-nvidia-gpu.json

```diff
@@ -6,5 +6,11 @@
   "GPU Power usage": "GPU Power usage",
   "GPU Encoder/Decoder": "GPU Encoder/Decoder",
   "GPUs": "GPUs",
-  "GPUs health": "GPUs health"
+  "GPUs health": "GPUs health",
+  "GPU providers": "GPU providers",
+  "NVIDIA": "NVIDIA",
+  "AMD": "AMD",
+  "other": "other",
+  " and ": " and ",
```

Comment:
we should not be doing any sentence constructions which is very bad for a correct translations.
So you need to specify sentences for all cases like
```
"Nvidia and AMD": "Nvidia and AMD"
"Nvidia and other": "Nvidia and other"
```
etc

Reply:
done